packages/lib/slots.ts

```diff
@@ -66,19 +62,24 @@ function buildSlotsWithDateRanges({
   const startTimeWithMinNotice = dayjs.utc().add(minimumBookingNotice, "minute");
 
   const orderedDateRanges = dateRanges.sort((a, b) => a.start.valueOf() - b.start.valueOf());
-  orderedDateRanges.forEach((range) => {
-    const dateYYYYMMDD = range.start.format("YYYY-MM-DD");
+  orderedDateRanges.forEach((range, i) => {
+    // avoids calling .format("YYYY-MM-DD") - which is a slow operation.
```

Comment:
.format takes about 2ms, this new way takes 0.006ms - micro improvement but these things add up.